pom.xml

```diff
@@ -94,7 +94,7 @@
         <antlr.version>4.7.1</antlr.version>
         <csv.version>1.4</csv.version>
         <lang3.version>3.3.1</lang3.version>
-        <guava.version>24.0-jre</guava.version>
+        <guava.version>24.1.1-jre</guava.version>
```

Comment:
We have to bump this up further to a 30.x version on the 5.3.x branch, so we should probably do the same here (or we just pint merge the [5.3.x PR](https://github.com/confluentinc/ksql/pull/7463/files) to sync the version all the way up to master)
@wcarlson5 do you think your PR is ready to be merged, or are you still investigating the other CVEs?

(also, why is 5.3.x already on 24.1.1, but master is only on 24.0 🤔 )

Reply:
I still have other CVEs I need to take care of but I don't think there is any reason to fix them all in the same PR. If you want to pint merge i don't see any problems as long as it works

Reply:
I updated it to 30.0-jre.

Reply:
Oh, I just realized this PR was against 5.2.x -- not sure why I thought it was master 🤦‍♀️ 

Anyways I think we should be good if we just bump it to `30.1.1-jre` -- then we can pint merge this PR instead of the 5.3.x one to get the `guava` version settled all the way up to master